nagios.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+)
+
+// NagiosStatusData is a task factory for tasks that fetch a JSON representation of
+// the provided project. The task uses outFor and errOutFor to get io.Writers
+// to write, respectively, the JSON output and any eventual error message.
+func GetNagiosStatusData(project string, outFor, errOutFor projectResourceWriterCloserFactory) Task {
+	return retrieveNagiosData(func(project, pod string) *exec.Cmd {
+		return exec.Command("oc", "exec", pod, "--", "cat", "/var/log/nagios/status.dat")
+	}, project, "Status", outFor, errOutFor)
+}
+
+// NagiosStatusData is a task factory for tasks that fetch a JSON representation of
+// the provided project. The task uses outFor and errOutFor to get io.Writers
+// to write, respectively, the JSON output and any eventual error message.
+func GetNagiosHistoricalData(project string, outFor, errOutFor projectResourceWriterCloserFactory) Task {
+	return retrieveNagiosData(func(project, pod string) *exec.Cmd {
+		return exec.Command("oc", "exec", pod, "--", "tar", "-c", "-C", "/var/log/nagios", "archives", "--to-stdout")
```

Comment:
Do we want to keep the full path in the archive?

If we would untar the output, would we want it to be in somewhere like
a. `rhmap-dumps/TIMESTAMP/nagios/var/log/nagios/...`; or
b. `rhmap-dumps/TIMESTAMP/nagios/...`

a or b?